datastore/src/main/java/io/spine/server/storage/datastore/DsStandStorage.java

```diff
@@ -75,7 +75,8 @@ public boolean delete(AggregateStateId id) {
     @Nullable
```

Comment:
I think it's not `@Nullable`. As long as you modify this method, please pay attention to things like that.